### PR TITLE
Updated a few rulesets

### PR DESCRIPTION
**Pull request by lqf96**
including:
- jizz.ppy.sh
- how-old.net
- taobao.com
- tmall.com

### CODE REVIEW

**Comment by donced** [comment]
FYI, taobao.com & tmall.com have been updated (https://github.com/EFForg/https-everywhere/commit/697829e0bd70e7809a551e516e9faf4da9fb9956 https://github.com/EFForg/https-everywhere/commit/f45939a8832f17a01871edce2d494065d7bbaa45)

File: src/chrome/content/rules/Taobao.xml [hunk 1]
```diff
@@ -1,25 +1,14 @@
 <!--
```
Comment (2d1) [inline src/chrome/content/rules/Taobao.xml]: Please use git revert, or create a new branch and cherry-pick the other commits. Removing whitespace in general is not helpful for readability.

File: src/chrome/content/rules/how-old.net.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<ruleset name="how-old.net">
```
Comment (2d1) [inline src/chrome/content/rules/how-old.net.xml]: This ruleset seems fine.

File: src/chrome/content/rules/ppy.sh.xml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <ruleset name="ppy.sh">
 	<target host="osu.ppy.sh" />
-	
+	<target host="jizz.ppy.sh" />
```
Comment (2d1) [inline src/chrome/content/rules/ppy.sh.xml]: This addition seems fine.

**Comment by semenko** [comment]
Looks like our tests are failing on this PR -- and it hasn't been updated in a while. I'm going to close this for now, but if you'd like it merged, feel free to re-open. Thanks! :)

**Pull request closed without merging**